sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkConfiguration.java

```diff
@@ -42,6 +42,9 @@
   /** Where events come from. */
   @JsonProperty public NexmarkUtils.SourceType sourceType = NexmarkUtils.SourceType.DIRECT;
 
+  /** If {@literal true}, only generate events and write them to local file. */
+  @JsonProperty public boolean generateInputFileOnly = false;
```

Comment:
It may help to allow user to specify a file path

Reply:
Done